### PR TITLE
update meson

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: subprojects/mir-core.wrap [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory=mir-core
+url=https://github.com/libmir/mir-core.git
+revision=head
```
Comment (9il) [inline subprojects/mir-core.wrap:4]: A project that will use stdx-allocator would need to override `wrap` file anyway.

**Comment by s-ludwig** [comment]
Looks like this needs a rebase to trigger Travis again.

**Comment by 9il** [comment]
> Looks like this needs a rebase to trigger Travis again.

looks like it does not help

File: .travis.yml [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+d:
+ - ldc
+ - ldc-beta
+ - dmd
+ - dmd-beta
+
+env:
+ - ARCH="x86_64"
+
+ matrix:
```
Comment (s-ludwig) [inline .travis.yml]: Can it be this leading space then, that produces a parsing error? Not sure how exactly YAML handles indentation.

Comment (9il) [reply .travis.yml]: Thanks!

**Comment by 9il** [comment]
passes

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
By the way i see that you pushed your branch to the origin but it's better to make PR with your fork because otherwise there are two CI tests (push and PR), which means computing resource wasting.
Afaik there's no way to avoid that in the settings.

**Comment by wilzbach** [comment]
> By the way i see that you pushed your branch to the origin but it's better to make PR with your fork because otherwise there are two CI tests (push and PR), which means computing resource wasting.
> Afaik there's no way to avoid that in the settings.

The trick is to instruct Travis only to build the `master` branch: https://github.com/dlang-community/stdx-allocator/pull/19

**Pull request merged**